std/json.d

```diff
@@ -344,7 +344,7 @@ struct JSONValue
         else static if (is(T : string))
         {
             type_tag = JSON_TYPE.STRING;
-            store.str = arg;
+            () @trusted { store.str = arg; }();
```

Comment:
Still unsafe, because the conversion from T to string might invoke a `@system` `alias this`-ed function.